scripts/changelog-ci.py

```diff
@@ -262,22 +263,31 @@ def _get_pull_requests_beetween_tags(self, start, end):
 
         return items
 
-    def _parse_data(self, version, pull_request_data):
+    def _parse_data(self, version, pull_request_data, is_point_release: bool):
         """Parse the pull requests data and return a writable data structure"""
         string_data = (
             '# ' + self.config['header_prefix'] + ' ' + version + '\n\n'
         )
 
         group_config = self.config['group_config']
-
-        if group_config:
+        items_string = ''
+
+        # If it's a point release, only grab things with the point release tag.
+        if is_point_release:
+            for pull_request in pull_request_data:
+                if POINT_RELEASE_LABEL in pull_request['labels']:
+                    items_string += self._get_changelog_line(pull_request)
+            if items_string:
+                if group_config:
+                    string_data += '\n#### ' + POINT_RELEASE_TITLE + '\n\n'
+                string_data += '\n' + items_string
+
+        elif (not is_point_release) and group_config:
```

Comment:
It looks like this elif could be simplified to just `elif group_config:`. Since the new `if` block executes when `is_point_release == True`, it must be `False` when this elif condition is checked. The 2nd `elif:` on 314 can also still work the same written as the original `else:`, as we'd only hit that if both resolved to False.

```suggestion
        elif group_config:
```